apps/web/cypress/integration/outcome/outcomes.spec.ts

```diff
@@ -55,17 +54,6 @@ describe('Recommendations', () => {
     cy.contains(questions.recommendation.call811.title);
   });
 
-  it('shows symptomatic & no test recommendation with secondary symptoms', () => {
-    cy.contains(questions.q1.question).no().continue();
-    cy.contains(questions.q2.question).no().continue();
-    cy.contains(questions.q3.question).selectSymptom('fever').continue();
-    answerNoForHealthWorkQuestions();
-    cy.contains(questions.hwq6.question).selectRadio('Partial1Dose').continue();
-    cy.contains(questions.hwq7.question).selectRadio('OverSeventy').continue();
-    cy.contains(questions.hwq8s.question).no().continue();
-    cy.contains(questions.recommendation.symptomaticNoNeedTest.title);
-  });
-
   it('shows symptomatic & no test recommendation with primary symptoms', () => {
```

Comment:
The final call is `symptomaticNeedTest` here, does this test name need updating?

Reply:
Yeah, I think the test needs updating. I will change it to fit current requirements